docs/content/concepts/ops-jobs-graphs/op-events.mdx

```diff
@@ -127,17 +127,19 @@ from dagster import op, AssetMaterialization, Output, EventMetadata
 def my_metadata_materialization_op(context):
     df = read_df()
     remote_storage_path = persist_to_storage(df)
-    yield AssetMaterialization(
-        asset_key="my_dataset",
-        description="Persisted result to storage",
-        metadata={
-            "text_metadata": "Text-based metadata for this event",
-            "path": EventMetadata.path(remote_storage_path),
-            "dashboard_url": EventMetadata.url("http://mycoolsite.com/url_for_my_data"),
-            "size (bytes)": calculate_bytes(df),
-        },
+    context.log_event(
+        AssetMaterialization(
+            asset_key="my_dataset",
+            description="Persisted result to storage",
+            metadata={
+                "text_metadata": "Text-based metadata for this event",
+                "path": EventMetadata.path(remote_storage_path),
+                "dashboard_url": EventMetadata.url("http://mycoolsite.com/url_for_my_data"),
+                "size (bytes)": calculate_bytes(df),
+            },
+        )
     )
-    yield Output(remote_storage_path)
+    return remote_storage_path
 ```
 
 ### Expectation Results
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"endLine":243}} -->
<i class='graphite__hidden'>[Re: line 243]<br/><br/></i>

i think this would be the only place where context.log_event wont be applicable ?